### PR TITLE
update DES vocabs - obs-prop and unit-of-measure

### PR DESCRIPTION
**Pull request by broughd-qld**
Updates to the DES vocabularies for Observed Properties and units of measure (including quantity kinds).

Added new properties for oceanographic terms and PFAS.

### CODE REVIEW

**Comment by broughd-qld** [comment]
Observed properties validates against VocPub profile.
Quantity Kind and Unit of Measure have validation errors for SHOULD constraints only, regarding provenance. 

Suggest that the SHOULD constraint issues are accepted. The terms have rdfs:isDefinedBy included.

File: vocabularies-des/qesd-obs-props.ttl [hunk 1]
```diff
@@ -967,7 +981,7 @@
 <https://linked.data.gov.au/def/qld-obsprop/a9335d70-3fc6-5181-b8fb-d6f8c938aa3d>	
     a skos:Concept ;
     skos:prefLabel "TSS & Nutrients"@en ;	
-    skos:altLabel "TSS & Nutrients-WQI Grouping"@en ;	
+    skos:altLabel "TSS & Nutrients-WQI Grouping"@en ;	#Suggest we remove WQI from the label
```
Comment (GitVillain) [inline vocabularies-des/qesd-obs-props.ttl:984]: suggest "TSS & Nutrients"

File: vocabularies-des/qesd-obs-props.ttl [hunk 2]
```diff
@@ -8951,3 +8979,83 @@
...
+    skos:definition "Wave direction at the peak frequency"@en ;
+    skos:prefLabel "Peak wave direction, Dirp"@en ;
+    skos:altLabel "Dirp" ;
+    qudt:hasQuantityKind quantitykind:Temperature ;
+    skos:broader <https://linked.data.gov.au/def/qld-obsprop/5fb80b7c-b505-5960-853a-23c85c7cf472> ;
+    dcterms:created "2015-08-06T06:45:48Z"^^xsd:dateTime ;
+    dcterms:modified "2022-08-05T11:24:28"^^xsd:datetime ;
+.
+
+<http://pid.geoscience.gov.au/def/voc/ga/netcdfldtos/sea_surface_temperature>
```
Comment (GitVillain) [inline vocabularies-des/qesd-obs-props.ttl:9051]: invalid url

File: vocabularies-des/qesd-obs-props.ttl [hunk 3]
```diff
@@ -8951,3 +8979,83 @@
...
+    qudt:hasQuantityKind quantitykind:Temperature ;
+    skos:broader <https://linked.data.gov.au/def/qld-obsprop/5fb80b7c-b505-5960-853a-23c85c7cf472> ;
+    dcterms:created "2015-08-06T06:45:48Z"^^xsd:dateTime ;
+    dcterms:modified "2022-08-05T11:24:28"^^xsd:datetime ;
+.
+
+<http://pid.geoscience.gov.au/def/voc/ga/netcdfldtos/sea_surface_temperature> 
+    a skos:Concept ;
+    skos:inScheme <https://linked.data.gov.au/def/qld-obsprop> ;
+    rdfs:isDefinedBy <http://pid.geoscience.gov.au/def/voc/ga/netcdfldtos> ;
```
Comment (GitVillain) [inline vocabularies-des/qesd-obs-props.ttl:9054]: invalid URL

File: vocabularies-des/qesd-obs-props.ttl [hunk 4]
```diff
@@ -501,6 +502,21 @@
...
 . 
 
+<https://linked.data.gov.au/def/qld-obsprop/2ac0e547-d4ca-5167-a250-79bc59b4f452>
+   a skos:Concept ;
+    skos:topConceptOf <https://linked.data.gov.au/def/qld-obsprop> ;
+    skos:inScheme <https://linked.data.gov.au/def/qld-obsprop> ; 
+    rdfs:isDefinedBy <https://linked.data.gov.au/def/qld-obsprop> ;
+    skos:narrower 
+         <https://linked.data.gov.au/def/qld-obsprop/5fb80b7c-b505-5960-853a-23c85c7cf472> ,
+         <http://pid.geoscience.gov.au/def/voc/ga/netcdfldtos/sea_surface_temperature> ;
```
Comment (GitVillain) [inline vocabularies-des/qesd-obs-props.ttl:512]: invalid URL

**Review by GitVillain — COMMENTED** [review]
1 suggested label name change at 984
and numerous invalid URLs all starting with http://pid....

File: vocabularies-des/qesd-obs-props.ttl (hunk 1 above)
Comment (broughd-qld) [reply vocabularies-des/qesd-obs-props.ttl:984]: Fixed

File: vocabularies-des/qesd-obs-props.ttl (hunk 4 above)
Comment (broughd-qld) [reply vocabularies-des/qesd-obs-props.ttl:512]: The redirection is not working. When on the ARDC Research Vocabs Australia website, the term can be found and resolves internally.

File: vocabularies-des/qesd-obs-props.ttl (hunk 2 above)
Comment (broughd-qld) [reply vocabularies-des/qesd-obs-props.ttl:9051]: Resolves on RVA.

File: vocabularies-des/qesd-obs-props.ttl (hunk 3 above)
Comment (broughd-qld) [reply vocabularies-des/qesd-obs-props.ttl:9054]: Resolves on RVA

**Review by GitVillain — APPROVED** [review]
Looks good

**Review by KellyVance — APPROVED** [review]

**Pull request merged**